app/code/Magento/Sales/Block/Adminhtml/Order/Create/Header.php

```diff
@@ -76,7 +76,7 @@ protected function _getCreateOrderTitle()
         $customerId = $this->getCustomerId();
         $storeId = $this->getStoreId();
         $out = '';
-        if ($customerId && $storeId) {
+        if (!empty($customerId) && $storeId) {
```

Comment:
I think, the usage of `!empty` a condition is also redundant here.

Reply:
I have refactored the if condition